templates/partial/_get-started-linux.html

```diff
@@ -0,0 +1,14 @@
+<ol class="p-stepped-list u-no-margin--left">
+  <li class="p-stepped-list__item">
+    <h3 class="p-stepped-list__title">
+      Install Multipass on Linux
+    </h3>
+    <div class="p-stepped-list__content">
+      <p>
+        <code>sudo snap install multipass</code>
```

Comment:
Should we use the code-snippet pattern here. Looks strange to change style between install and usage?